receiver/http.go

```diff
@@ -192,6 +198,43 @@ func (htt *HTTP) Start() error {
 	return skogul.Error{Reason: "Shouldn't reach this"}
 }
 
+// verifyPeerCertificate verifies a client certificate presented to us
+// during TLS handshake by comparing its extensions (such as SAN) to
+// some expected value(s)
+func (htt *HTTP) verifyPeerCertificate(_ [][]byte, verifiedChains [][]*x509.Certificate) error {
+	certLogger := httpLog
```

Comment:
Logs in here should maybe be left out of the common path, i.e. opt-in through config? This could be a high throughput function which shouldn't incur any extra overhead unless requested.